src/dali/plugin/pair_converter/csv/kraken.py

```diff
@@ -59,13 +62,40 @@
 _MEDIA: str = "media"
 _QUERY: str = "q"
 
+# Time periods
 _MS_IN_SECOND: int = 1000
+_SECONDS_IN_DAY: int = 86400
+_CHUNK_SIZE: int = _SECONDS_IN_DAY * 30
+_SECONDS_IN_MINUTE: int = 60
+
+# Time in minutes
+_MINUTE: str = "1"
+_FIVE_MINUTE: str = "5"
+_FIFTEEN_MINUTE: str = "15"
+_ONE_HOUR: str = "60"
+_TWELVE_HOUR: str = "720"
+_ONE_DAY: str = "1440"
+_TIME_GRANULARITY: List[str] = [_MINUTE, _FIVE_MINUTE, _FIFTEEN_MINUTE, _ONE_HOUR, _TWELVE_HOUR, _ONE_DAY]
+
+# Chunking variables
+_PAIR_START: str = "start"
+_PAIR_MIDDLE: str = "middle"
+_PAIR_END: str = "end"
+_MAX_MULTIPLIER: int = 500
+
+
+class PairStartEnd(NamedTuple):
```

Comment:
This looks like a private class: if so, prepend with `_`

Reply:
You meant protected right?

Reply:
Protected can only be attributed to members of a class. In this case we're referring to a class name which is in the global scope, so it can only be public or private wrt to the scope it's in.